pkg/service/labels/constants.go

```diff
@@ -0,0 +1,36 @@
+package labels
+
+import "fmt"
+
+type Label string
+
+const (
+	SpawnerLabel             = "spawner"
```

Comment:
Can we not call this Label as it is confusing to read along with other labels?

Reply:
changed and kept it as `Spawner`, so it will be like `constants.Spawner`